src/ad/azext_ad/azext_metadata.json

```diff
@@ -0,0 +1,3 @@
+{
+    "azext.minCliCoreVersion": "2.15.0"
```

Comment:
not in preview or experimental?

Reply:
`az ad` is stable. `az ad ds` is marked as preview in `command.py`